tests/bluetooth/tester/src/btp.c

```diff
@@ -356,3 +356,23 @@ void tester_rsp(uint8_t service, uint8_t opcode, uint8_t status)
 	(void)memset(cmd, 0, sizeof(*cmd));
 	k_fifo_put(&avail_queue, cmd);
 }
+
+void tester_supported_commands(uint8_t service, uint8_t *cmds, uint16_t *cmds_size)
```

Comment:
What is the rationale behind setting the `cmds_size` as a output pointer, rather than a return value?

Reply:
just to keep out parameters together,  I can use return value